docs/pipelineruns.md

```diff
@@ -160,8 +160,7 @@ You can reference a `Tekton bundle` in a `TaskRef` in both `v1` and `v1beta1` us
 In `v1beta1`, you can also reference a `Tekton bundle` using OCI bundle syntax, which has been deprecated in favor of remote resolution. The example shown below for `v1beta1` uses OCI bundle syntax, and requires enabling `enable-tekton-oci-bundles: "true"` feature flag.
 
 {{< tabs >}}
-{{< tab "v1 & v1beta1" >}}
-```yaml
+{{< tab header="v1 & v1beta1" code=true lang="yaml" >}}
```

Comment:
I'm curious why the ```yaml is removed here. When I display the rich diff, it no longer renders code blocks. Will this render correctly when people are viewing the docs on GitHub? And is there a way to verify that it will render correctly on the website?